src/devices/MotorHat/samples/MotorHat.Sample.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using MotorHat.Samples;
+using System;
+
+namespace Iot.Device.MotorHat.Samples
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Sample for each device separately
+
+            DCMotorSample.Run();
+            // StepperMotorSample.Run();
```

Comment:
remove comments in the Maine and move all the code here from the other file since there is not much value in having this separate